volume/local/local_unix.go

```diff
@@ -27,6 +27,9 @@ var (
 		"o":      true, // generic mount options
 		"device": true, // device to mount from
 	}
+	mandatoryKey = map[string]struct{}{
+		"device": {},
```

Comment:
Discussing with @cpuguy83 and we should add `type` to this list as well; can you add that?